src/Symfony/Bridge/Twig/Resources/views/Form/foundation_5_layout.html.twig

```diff
@@ -165,21 +165,25 @@
 {%- endblock choice_widget_collapsed %}
 
 {% block choice_widget_expanded -%}
-    {% if '-inline' in label_attr.class|default('') %}
+    {% if '-inline' in label_attr.class|default('') -%}
```

Comment:
@HeahDude it's not related to this PR so should be done in a separate one. It should also be done on the latest branch this layout is available, to ease future merging.